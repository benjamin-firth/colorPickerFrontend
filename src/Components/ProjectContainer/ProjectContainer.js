import React from 'react';
import './ProjectContainer.scss';
import Project from '../Project/Project';
import PropTypes from 'prop-types';

function ProjectContainer() {
  return (
    <section>
    this is a project container
      <Project />
      <Project />
    </section>
  );
}

export default ProjectContainer;
