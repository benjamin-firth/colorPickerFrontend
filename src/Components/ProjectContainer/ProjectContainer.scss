.projects-container {
  column-count: 2;
  column-gap: 4vw;
  padding: 1vw;
}
