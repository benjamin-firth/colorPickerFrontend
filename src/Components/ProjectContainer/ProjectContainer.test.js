import React from 'react';
import { render } from '@testing-library/react';
import ProjectContainer from './ProjectContainer';
import { shallow } from 'enzyme';

jest.mock("react-redux", () => ({
  useSelector: () => [{name: "Test", id: 1}],
  useDispatch: () => jest.fn()
}));

describe('ProjectContainer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ProjectContainer />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
