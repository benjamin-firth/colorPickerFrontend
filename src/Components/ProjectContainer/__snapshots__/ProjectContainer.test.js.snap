// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectContainer should match the snapshot 1`] = `
<section
  className="projects-container"
>
  <h2>
    Loading...
  </h2>
</section>
`;
